docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.install

```diff
@@ -1026,3 +1027,90 @@ function su_humsci_profile_update_9714() {
     $role->save();
   }
 }
+
+/**
+ * Remove any existing hs_webform paragraph entities.
+ */
+function su_humsci_profile_update_9715($sandbox) {
+  $paragraph_storage = \Drupal::entityTypeManager()->getStorage('paragraph');
+  $paragraph_machine_name = 'hs_webform';
+
+  if (empty($sandbox['ids'])) {
+    $sandbox['ids'] = $paragraph_storage->getQuery()
+      ->condition('type', $paragraph_machine_name)
+      ->accessCheck(FALSE)
+      ->execute();
+    $sandbox['total'] = count($sandbox['ids']);
+  }
+  $paragraph_ids = array_splice($sandbox['ids'], 0, 50);
+
+  foreach ($paragraph_storage->loadMultiple($paragraph_ids) as $paragraph) {
+    $paragraph->delete();
+  }
+
+  $sandbox['#finished'] = count($sandbox['ids']) ? 1 - count($sandbox['ids']) / $sandbox['total'] : 1;
+}
+
+/**
+ * Delete hs_webform paragraph type and remove webform config.
+ */
+function su_humsci_profile_update_9716() {
+  $paragraph_machine_name = 'hs_webform';
+  // Delete the field_hs_webform field storage (and field instances).
+  FieldStorageConfig::load('field_hs_webform')?->delete();
+
+  // Delete the paragraph itself.
+  ParagraphsType::load($paragraph_machine_name)?->delete();
+
+  // Remove hs_webform from config ignored configs.
+  $config_storage = \Drupal::service('config.storage.sync');
+
+  $configs_to_change = [
+    'node.hs_basic_page.field_hs_page_components' => 'node.field_hs_page_components',
+    'node.hs_basic_page.field_hs_page_hero' => 'node.field_hs_page_hero',
+    'node.hs_private_page.field_hs_priv_page_components' => 'node.field_hs_priv_page_components',
+  ];
+
+  /** @var \Drupal\Core\Entity\EntityStorageInterface $field_config_storage */
+  $field_config_storage = \Drupal::service('entity_type.manager')->getStorage('field_config');
+
+  /** @var \Drupal\Core\Entity\EntityStorageInterface $fieldConfigStorage */
+  $field_storage = \Drupal::service('entity_type.manager')->getStorage('field_storage_config');
+
+  foreach ($configs_to_change as $field_config => $field_storage_config) {
+
+    // If storage does not yet exist, create it.
+    if (empty($field_storage->load($field_storage_config))) {
+      $field_storage->createFromStorageRecord($config_storage->read('field.storage.' . $field_storage_config))->save();
+    }
+
+    // Update the field instances from filesystem.
+    $config = $field_config_storage->load($field_config);
+    if (empty($config)) {
+      $field_config_storage->create($config_storage->read('field.field.' . $field_config))->save();
+    }
+  }
+
+  // Remove webform existing config - needs to be done before uninstall.
+  $config_factory = \Drupal::configFactory();
+
+  $config_names = array_filter($config_factory->listAll(), function ($item) {
+    return str_contains($item, 'webform');
+  });
+
+  foreach ($config_names as $config) {
+    $config_factory->getEditable($config)->delete();
+    $message = t('Deleted @config', [
+      '@config' => $config,
+    ]);
+    Drush::logger()->notice($message);
```

Comment:
Replace with normal logger or messenger. Drush shoudn't be used in update hooks like this.